### PR TITLE
simplify flipper settings

### LINKED ISSUE #828: Simplify start settings
New users commonly get confused/overwhelmed by the settings options. 
They should be simplified or restructured.
There should be a simple toggle that switches between lbin and median.

### PR DESCRIPTION
**Pull request by matthias-luger**
fixes #828

### CODE REVIEW

**Comment by matthias-luger** [comment]
@Ekwav Could you maybe check this out and just take a quick look at it, if this is what you had in mind?

**Comment by matthias-luger** [comment]
@Ekwav bump

File: components/Flipper/FlipperFilter/FlipperFilter.tsx [hunk 1]
```diff
@@ -81,84 +62,57 @@ function FlipperFilter(props: Props) {
...
-    function onFreePremiumComplete() {
-        setFreePremiumFilters(true)
+    function numberFieldMaxValue(value: number = 0, maxValue: number) {
+        return value <= maxValue
     }
 
-    function onFreeLoginComplete() {
-        setFreeLoginFilters(true)
+    function isCurrentCalculationBasedOnLbin() {
+        return flipCustomizeSettings.finders?.length === 1 && flipCustomizeSettings.finders[0] === 2
```
Comment (Ekwav) [inline components/Flipper/FlipperFilter/FlipperFilter.tsx]: I am not sure why but this doesn't work, when selecting only the `sniper` in `settings` it still displays median

File: components/Flipper/FlipperFilter/FlipperFilter.tsx [hunk 2]
```diff
@@ -81,84 +62,57 @@ function FlipperFilter(props: Props) {
...
+                category: 'customizeFlipStyle',
+                action: 'finders: 1,4'
+            })
+            trackEvent({
+                category: 'customizeFlipStyle',
+                action: 'lbin: false'
+            })
+        } else {
+            flipCustomizeSettings.finders = [2]
+            flipCustomizeSettings.useLowestBinForProfit = true
```
Comment (Ekwav) [inline components/Flipper/FlipperFilter/FlipperFilter.tsx:95]:
```suggestion
```
that setting is deprecated, please avoid activating it

File: components/Flipper/FlipperFilter/FlipperFilter.tsx [hunk 3]
```diff
@@ -179,170 +133,262 @@ function FlipperFilter(props: Props) {
...
+                        </Form.Group>
+                        <Form.Group className={styles.filterTextfield}>
+                            <Tooltip
+                                type="hover"
+                                content={
+                                    <Form.Label htmlFor="min-volume" className={`${styles.flipperFilterFormfieldLabel} ${styles.checkboxLabel}`}>
+                                        Min. Volume:
+                                    </Form.Label>
+                                }
+                                tooltipContent={<span>Minimum amount of sold auctions in the last 24 hours</span>}
```
Comment (Ekwav) [inline components/Flipper/FlipperFilter/FlipperFilter.tsx]:
```suggestion
                                tooltipContent={<span>Minimum average amount of sells in 24 hours</span>}
```
its mostly not the last 24 hours but longer/shorter timespans

File: components/Flipper/FlipperFilter/FlipperFilter.tsx [hunk 4]
```diff
@@ -179,170 +133,262 @@ function FlipperFilter(props: Props) {
...
+                    onClick={() => {
+                        setShowCustomizeFlip(true)
+                    }}
+                    className={styles.filterCheckbox}
+                >
+                    <span className={styles.filterBorder}>
+                        <Tooltip
+                            type="hover"
+                            content={<span style={{ cursor: 'pointer', marginRight: '10px' }}>Settings</span>}
+                            tooltipContent={<span>Edit flip appaerance and general settings</span>}
```
Comment (Ekwav) [inline components/Flipper/FlipperFilter/FlipperFilter.tsx]:
```suggestion
                            tooltipContent={<span>Edit flip appearance and general settings</span>}
```

File: components/Flipper/FlipperFilter/FlipperFilter.tsx [hunk 5]
```diff
@@ -179,170 +133,262 @@ function FlipperFilter(props: Props) {
...
+                                htmlFor="onlyBinCheckbox"
+                                className={`${styles.flipperFilterFormfieldLabel} ${styles.checkboxLabel}`}
+                                defaultChecked={flipperFilter.onlyBin}
+                            >
+                                Only BIN-Auctions
+                            </Form.Label>
+                        }
+                        tooltipContent={
+                            flipperFilter.onlyBin ? (
+                                <span>Dont show auction flips that are about to end and could be profited from with the current bid</span>
```
Comment (Ekwav) [inline components/Flipper/FlipperFilter/FlipperFilter.tsx]:
```suggestion
                                <span>Do not show auction flips that are about to end and could be profited from with the current bid</span>
```

File: components/Flipper/FlipperFilter/FlipperFilter.tsx [hunk 6]
```diff
@@ -179,170 +133,262 @@ function FlipperFilter(props: Props) {
...
+                    onClick={() => {
+                        setShowRestrictionList(true)
+                    }}
+                    className={styles.filterCheckbox}
+                    style={{ cursor: 'pointer' }}
+                >
+                    <span className={styles.filterBorder}>
+                        <Tooltip
+                            type="hover"
+                            content={<span className={`${styles.flipperFilterFormfieldLabel} ${styles.checkboxLabel}`}>Blacklist</span>}
```
Comment (Ekwav) [inline components/Flipper/FlipperFilter/FlipperFilter.tsx]:
```suggestion
                            content={<span className={`${styles.flipperFilterFormfieldLabel} ${styles.checkboxLabel}`}>Filter-Rules</span>}
```
whitelists are often overlooked

Comment (Ekwav) [inline components/Flipper/FlipperFilter/FlipperFilter.tsx]:
```suggestion
                            content={<span className={`${styles.flipperFilterFormfieldLabel} ${styles.checkboxLabel}`}>Filter-Rules</span>}
```
whitelists are often overlooked

**Review by Ekwav — CHANGES_REQUESTED** [review]

**Review by Ekwav — APPROVED** [review]
LGTM!

**Pull request merged**